web_accounts_list.json

```diff
@@ -806,7 +806,17 @@
       "account_missing_code": "404",
       "known_accounts": ["test", "test2"],
       "category": "misc",
-      "valid": true
+      "valid": true
     },
```

Comment:
Can you check on this line? Looks like there is a `^M` (Ctrl-M) in there. Please confirm or deny.

Reply:
@WebBreacher

Answer= `deny`
I saw that yesterday after checking the PR, I thought I needed to make another commit to get rid of it, but I didn't find this anywhere.

<img width="633"  src="https://user-images.githubusercontent.com/104733166/186476613-40f3bf1a-13b8-49f1-a02d-81c75f504522.png">

Reply:
@WebBreacher 
Found this:https://stackoverflow.com/questions/1822849/what-are-these-ms-that-keep-showing-up-in-my-files-in-emacs